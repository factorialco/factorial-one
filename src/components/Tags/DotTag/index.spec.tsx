import { render } from "@testing-library/react-native";
import React from "react";
import { DotTag, dotTagColors } from ".";

describe("RawTag", () => {
  it("Snapshot", () => {
    dotTagColors.map((color) => {
      const { toJSON } = render(
        <DotTag key={color} text="Label" color={color} />,
      );
      expect(toJSON()).toMatchSnapshot();
    });
  });
});
