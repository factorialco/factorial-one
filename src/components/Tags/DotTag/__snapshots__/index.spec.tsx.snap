// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RawTag Snapshot 1`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(0 0% 100% / 0.5)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 2`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(undefined)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 3`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(undefined)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 4`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(217 96% 11% / 0.61)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 5`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(340 49% 60%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 6`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(331 84% 63%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 7`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(192 26% 54%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 8`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(162 44% 33%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 9`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(84 55% 53%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 10`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(239 91% 64%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 11`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(25 46% 53%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 12`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(348 80% 50%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 13`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(184 92% 35%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 14`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(25 95% 53%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 15`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(5 100% 65%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 16`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(160 84% 39%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 17`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(216 90% 65%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 18`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(38 92% 54%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 19`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(258 88% 67%)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;

exports[`RawTag Snapshot 20`] = `
<View
  className="flex items-start"
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    className="flex flex-row items-center justify-start gap-0.5 rounded-full py-0.5 pr-2 pl-1 border border-solid border-f1-border-secondary"
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      aria-hidden={true}
      className="m-1 aspect-square w-2 rounded-full"
      style={
        {
          "backgroundColor": "hsl(undefined)",
        }
      }
    />
    <Text
      className="line-clamp-1 text-base font-medium text-f1-foreground"
    >
      Label
    </Text>
  </View>
</View>
`;
